client/src/components/OptionsDrawer/OptionsDrawer.tsx

```diff
@@ -3,6 +3,7 @@ import { Typography, ListItem, ListItemText, ListItemIcon, List, Divider, Drawer
 import SwapHorizontalCircleOutlinedIcon from '@material-ui/icons/SwapHorizontalCircleOutlined';
 import useStyles from './optionsDrawerStyles';
 import { useKanban } from '../../context/';
+import { useHistory } from 'react-router-dom';
```

Comment:
Put this import above our app's files; below `@material-ui` import but above `useStyles`

Reply:
Sure, thanks!